magento/adobe-stock-integration
# Quantity of images is not updated after image was licensed && License and Save popup must be closed after confirm click immediately

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#826:Quantity of images on account profile is not updated after image was 
2. magento/adobe-stock-integration#852:"License and Save" button is not protected from multiple clicks

### Manual testing scenarios (*)

  

1.   Login to admin panel
2.     Navigate to Cms Pages
3.     User clicks Add New Page button
4.     Expand "Content" section
5.     User clicks "Insert Image..." button
6.     Click "Search Adobe Stock" button to open images grid
7.     Sign into Adobe IMS
8.     Check quantity of images on your account
9.     Expand preview image of any not licensed image
10.     Click License and Save
11.     Click "Search Adobe Stock" button to open images grid again
12.     Check quantity of images on your account

Expected result (*)

Quantity of images is updated(decrease by 1 )
